automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -272,4 +274,18 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     testResult.get
   }
 
+  def withNewGoogleBucket[T](googleProject: GoogleProject)(testCode: GcsBucketName => T): T = {
```

Comment:
Should we have the option here to specify a name, like cluster create does?

Reply:
yep